package.json

```
@@ -2,7 +2,7 @@
   "name": "postcss-preset-mantine",
   "version": "1.3.0",
   "description": "PostCSS preset for Mantine (7.0+) applications",
-  "main": "preset.js",
+  "main": "preset.ts",
```

```suggestion
  "main": "dist/preset.js",
```
So I just need to change it like that? @rtivital

---

yes, also make sure that dist folder is in gitignore